src/components/ProgressIndicator/_progress-indicator.scss

```
@@ -1,5 +1,36 @@
 @import '~carbon-components/scss/components/progress-indicator/progress-indicator';
+@import '../../globals/vars';
 
-.bx--skeleton .bx--progress-step {
-  flex: unset;
+.#{$iot-prefix}--progress-indicator {
+  .bx--skeleton .bx--progress-step {
```

Same thing here with the hardcoded prefix:
 ```suggestion
  .#{prefix}--skeleton .#{prefix}--progress-step {
```